RoaringBitmap/src/main/java/org/roaringbitmap/ArrayContainer.java

```diff
@@ -1125,10 +1125,20 @@ public int serializedSizeInBytes() {
   @Override
   public BitmapContainer toBitmapContainer() {
     BitmapContainer bc = new BitmapContainer();
-    bc.loadData(this);
+    copyBitmapTo(bc.bitmap, 0);
+    bc.cardinality = cardinality;
     return bc;
   }
 
+  @Override
+  public void copyBitmapTo(long[] dest, int position) {
+    for (int k = 0; k < cardinality; ++k) {
+      final char x = content[k];
+      final int i = position + (x) / 64;
+      dest[i] |= (1L << x);
+    }
```

Comment:
This is unnecessarily complicated.

```
   data[position + x/64] |= 1L << x;
```

ought to be enough. There is no need for two lines.